beetsplug/missing.py

```diff
@@ -252,10 +252,23 @@ def _missing_albums(self, lib, query):
             if total:
                 continue
 
-            for album in missing:
-                for album_info in hooks.albums_for_id(album['id']):
-                    item = _album_item(album_info, album_info.album_id)
-                    print_(format(item, fmt))
+            for release_group in missing:
+                try:
+                    releases_object = musicbrainzngs.browse_releases(
+                            release_group=release_group['id']
+                        )
+                    albums_list = releases_object['release-list']
+                except MusicBrainzError as err:
+                    self._log.info(
+                        u"Couldn't fetch info for release group '{}' - '{}'",
+                        release_group['id'], err
+                    )
+                    continue
+
+                for album in albums_list:
+                    for album_info in hooks.albums_for_id(album['id']):
+                        item = _album_item(album_info, album_info.album_id)
+                        print_(format(item, fmt))
```

Comment:
So, to `print_` `format`, I need an item created by the filter at the top of the page, but the original object in the original code was the output of `musicbrainzngs.browse_release_groups`, these objects didn't contain any of the required attributes.
So for every missing release group I get the list of missing albums per release group.
This way I now have all the variables and can use the `format` function using all the placeholder already defined in the documentation.